andino_bringup/config/camera_params.yaml

```diff
@@ -0,0 +1,6 @@
+v4l2_camera_node:
+  ros__parameters:
+    image_size: [640,480]
+    camera_frame_id: "camera_link"
+    camera_info_url: file:///home/jesus-silva/ws/src/andino/andino_bringup/config/raspicam.yaml
```

Comment:
Note this path local to your laptop

Reply:
I didn't find a way to add it as relative path in the YAML file, because you need the 'file://' part, so I added it as a launch argument.